doc/reference/configuration/configuration_reference.rst

```diff
@@ -4767,6 +4768,37 @@ To learn more about the WAL configuration, check the :ref:`Persistence <configur
     | Default: 16777216
     | Environment variable: TT_WAL_QUEUE_MAX_SIZE
 
+..  _configuration_reference_wal_retention_period:
+
+..  confval:: wal.retention_period
+
+    **Since:** :doc:`3.1.0 </release/3.1.0>` (`Enterprise Edition <https://www.tarantool.io/compare/>`_ only)
+
+    The delay in seconds used to prevent the :ref:`Tarantool garbage collector <configuration_persistence_checkpoint_daemon>` from removing a :ref:`write-ahead log <internals-wal>` file after it has been closed.
+    If a node is restarted, ``wal.retention_period`` counts down from the last modification time of the write-ahead log file.
+
+    Setting up ``wal.retention_period`` may be useful for:
+
+    *   Anonymous replicas (see :ref:`replication.anon <configuration_reference_replication_anon>`).
+    *   CDC (Change Data Capture) that retrieves data using anonymous replication.
+
+    For example, a rebootstrap may be needed in the event of a downtime in a replica or the CDC.
```

Comment:
Which happens, as GC doesn't save xlog files needed for anonymous replicas. Up to you

In [checkpoint-daemon-garbage-collector](https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-checkpoint-daemon-garbage-collector) we say:

> Tarantool garbage collector doesn’t delete a file in the following cases:
> * A replica has fallen behind. The progress of each replica is tracked; if a replica’s position is far from being up to date, then the server stops to give it a chance to catch up. If an administrator concludes that a replica is permanently down, then the correct procedure is to restart the server, or (preferably) [remove the replica from the cluster](https://www.tarantool.io/en/doc/latest/how-to/replication/repl_bootstrap/#replication-remove-instances).

This doesn't apply to anonymous replicas